static/js/actions/financial_aid.js

```diff
@@ -16,7 +16,9 @@ export const CLEAR_CALCULATOR_EDIT = 'CLEAR_CALCULATOR_EDIT';
 export const clearCalculatorEdit = createAction(CLEAR_CALCULATOR_EDIT);
 
 export const UPDATE_CALCULATOR_EDIT = 'UPDATE_CALCULATOR_EDIT';
-export const updateCalculatorEdit = createAction(UPDATE_CALCULATOR_EDIT);
+export const updateCalculatorEdit = createAction(UPDATE_CALCULATOR_EDIT, 
+  option => option.value || option
```

Comment:
I'm fine either way, this was changed because `react-select` calls the `onChange` function with the new selected option. We can change it to the call site if you think that's cleaner (I kinda agree!).